deploy/charts/arcadia/templates/rag_job_rbac.yaml

```diff
@@ -53,13 +53,6 @@ rules:
   verbs:
   - list
   - get
-- apiGroups:
```

Comment:
@0xff-dev  So we do not require this permission anymore ?  Even we fix it to the correct group

Reply:
This permission is needed, I had the wrong group for the prompt resource. There's also a previous entry about prompt's permissions that fulfills the requirement, so just remove it here.

Reply:
please show me the link(the previous entry)

Reply:
https://github.com/kubeagi/arcadia/blob/9d3ee31434dc0e587fca1392a336fb2e945228af/deploy/charts/arcadia/templates/rag_job_rbac.yaml#L43